src/collective/fingerpointing/logger.py

```diff
@@ -12,32 +14,33 @@
 FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
 
 # by default, the audit log will use the same location used for the event log
-eventlog = getattr(getConfiguration(), 'eventlog', None)
+zopeConf = getConfiguration()
 
-if eventlog is None:
-    # we are running tests
-    logfile = os.path.join('.', AUDITLOG)
-else:
-    try:
-        logpath = eventlog.handler_factories[0].instance.baseFilename
-        logfolder = os.path.split(logpath)[0]
-        logfile = os.path.join(logfolder, AUDITLOG)
-    except AttributeError:
-        # we are in the debug console
-        logfile = os.path.join('.', AUDITLOG)
+auditlogConf = getattr(
+    zopeConf,
+    'product_config',
+    {}
+).get(
+    'collective.fingerpointing',
+    {}
+)
 
+logfile = auditlogConf.get('audit-log', './audit.log')
```

Comment:
tests fail because we have to include `<product-config collective.fingerpointing>` on Zope configuration before running the to create out `audit.log` file the way is supposed to work.

I think there's a Zope directive to add a configuration string, but I don't remember which one.